### ISSUE TITLE
Cannot view AAD-enabled Linux cluster nodes + apps in Azure portal

### ISSUE THREAD

**Issue by DianeHadley** [microsoft/service-fabric#399]
At this time, if you have an AAD-enabled Linux cluster in Azure, you will not be able to view the nodes or applications in Azure Portal:

![image](https://user-images.githubusercontent.com/43074866/66230402-f5840c00-e698-11e9-8315-ef56e64e7030.png)

We do not currently have a date by which this is expected to be fixed.

**Comment by johncrim** [comment]
Thanks for adding this to GitHub. I reported this issue to Azure support in June. It's important to note that it is recommended to secure Service Fabric clusters hosted in Azure with AAD:
https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-creation-setup-aad

So this failure should be considered a "normal" use case.

**Comment by DianeHadley** [comment]
@johncrim Yes, this is definitely a bug in the normal use case. It's a result of a difference between the frameworks we use for Linux and Windows. We are working on a way around it. Thanks for helping bring it to our attention :)

**Comment by christopherpcosta** [comment]
Automated deployments using SFCTL (using Jenkins for example) are also identified to be hit by this issue when AAD is enabled. (msft internal track id 174982952).

**Comment by vineet761** [comment]
Hi Team- Is there any update on this issue ?

**Comment by tg123** [comment]
we are working on it

**Comment by fuocor** [comment]
Any updates

**Comment by ravgupms** [comment]
@tg123 any update?

**Comment by cata** [comment]
Hi, 
I was wondering if you have any updates on this issue.

**Comment by TheAngryByrd** [comment]
Any updates here?
